File: app/src/main/java/com/example/mettest/models/Item.java

package com.example.mettest.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Item {

    @SerializedName("product_id")
    @Expose
    public String productId;
    @SerializedName("product_suk_id")
    @Expose
    public String productSukId;
    @SerializedName("product_name")
    @Expose
    public String productName;
    @SerializedName("category_id")
    @Expose
    public String categoryId;
    @SerializedName("description")
    @Expose
    public String description;
    @SerializedName("product_image")
    @Expose
    public String productImage;
    @SerializedName("price")
    @Expose
    public String price;
    @SerializedName("vendor_id")
    @Expose
    public String vendorId;
    @SerializedName("status")
    @Expose
    public String status;
    @SerializedName("created_at")
    @Expose
    public String createdAt;
    @SerializedName("updated_at")
    @Expose
    public String updatedAt;
    @SerializedName("category_name")
    @Expose
    public String categoryName;
    @SerializedName("vendor_name")
    @Expose
    public String vendorName;

}

File: app/src/main/java/com/example/mettest/models/Item.kt
package com.example.mettest.models

import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose
import com.example.mettest.models.Datum

class Item {
    @SerializedName("product_id")
    @Expose
    var productId: String? = null

    @SerializedName("product_suk_id")
    @Expose
    var productSukId: String? = null

    @JvmField
    @SerializedName("product_name")
    @Expose
    var productName: String? = null

    @SerializedName("category_id")
    @Expose
    var categoryId: String? = null

    @SerializedName("description")
    @Expose
    var description: String? = null

    @JvmField
    @SerializedName("product_image")
    @Expose
    var productImage: String? = null

    @JvmField
    @SerializedName("price")
    @Expose
    var price: String? = null

    @SerializedName("vendor_id")
    @Expose
    var vendorId: String? = null

    @SerializedName("status")
    @Expose
    var status: String? = null

    @SerializedName("created_at")
    @Expose
    var createdAt: String? = null

    @SerializedName("updated_at")
    @Expose
    var updatedAt: String? = null

    @JvmField
    @SerializedName("category_name")
    @Expose
    var categoryName: String? = null

    @SerializedName("vendor_name")
    @Expose
    var vendorName: String? = null
}